src/components/Footer/js/footerLinks.js

```diff
@@ -1,6 +1,62 @@
+import projectCards from '../../../pages/js/projects.js';
+
+//  static data can be generated from the projects.js file
+// [
+//   {
+//     label: 'Oneleif',
+//     path: '/projects',
+//     internal: true
+//   },
+//   {
+//     label: 'Bread for the Journey',
+//     path: '/projects',
+//     internal: true
+//   },
+//   {
+//     label: 'Swift UI Kit',
+//     path: '/projects',
+//     internal: true
+//   },
+//   {
+//     label: 'Leifdown',
+//     path: '/projects',
+//     internal: true
+//   },
+//   {
+//     label: 'Filodex',
+//     path: '/projects',
+//     internal: true
+//   },
+//   {
+//     label: 'Onefit',
+//     path: '/projects',
+//     internal: true
+//   }
+// ]
+
+/************************************
+ * Proposal
+ ************************************/
+// the data can be generated creating the array of objects
+// pros: less code
+// cons: logic implementation if path and internal differ between objects
+
+const parseLinks = projectCards.map(project => {
```

Comment:
true at this point i like this idea, we can have it randomly select 3 projects each time .You should submit an issue to add that functionality. At some point we'd want it to load those links dynamically off the backend

Reply:
understood.